.github/ISSUE_TEMPLATE/05_unhandledexception.md

```diff
@@ -0,0 +1,38 @@
+---
+name: 😱 Unhandled Exception
+about: Reporting holes in our bug reporting system
+title: ''
+labels: bug
+---
+
+<!-- This is a template that helps us provide quicker feedback. Please use any relevant sections and delete what you don't need. -->
+
+### Issue Description
+
+### Steps to Reproduce
+<!--
+* Include as many of these as possible:
+* A minimal sample project that reproduces the issue
+* IDE or CLI steps that create the project and reproduce the issue
+* Your command line invocation
+
+If providing steps to reproduce the issue proves difficult, consider attaching a binlog:
+[Details on sharing binary logs](https://aka.ms/msbuild/binlog)
+[More information on binary logs](https://github.com/dotnet/msbuild/blob/main/documentation/wiki/Binary-Log.md)
+NOTE: Binary logs capture environment information. Ensure it does not include any information you don't want to be public before you attach one.
+If you want to share one just with Microsoft, you can [report a problem through Visual Studio](https://developercommunity.visualstudio.com/report) and share it privately with Microsoft.
```

Comment:
This is great as part of `Steps to Reproduce` rather than in its own section. Can you modify the other issues to remove the `Attach a Binlog` section in favor of combining it with `Steps to Reproduce`? More visibility hopefully means more binlogs in the initial bug report